tests/tests-api.php

```diff
@@ -253,12 +258,14 @@ public function test_get_queried_version() {
 
 		try {
 			self::$api->process_query();
-			$this->assertEquals( 'v1', self::$api->get_queried_version() );
+		} catch ( WPDieException $e ) {}
```

Comment:
All tests that have this change were due to:

`self::$api->process_query();` includes a `wp_die()` at the end, hence catching the exception. However, all the assertions were being made inside the `try { }` block. But the flow is:

- `process_query()`
- Triggers exception
- Therefore following assertions were never being made.

Moving the assertions outside the try/catch block ensures they actually do something. All these were "risky" tests on my end due to no assertions actually being made.